specs/ServerCertificate.md

```diff
@@ -0,0 +1,229 @@
+if (m_webview)
+{
+  CHECK_FAILURE(m_webView->ClearSSLDecision());
+  MessageBox(nullptr, L"Cleared", L"Clear SSL Decision", MB_OK);
+}
+```
+## . NET/ WinRT
+```c#
+// This example bypass default SSL interstitial page using ReceivingServerCertificateError
+// event handler and to continue with the SSL certificate that is signed by an authority
+// that WebView2 don't trust. Otherwise, cancel the request.
+private bool _isServerCertificateError = false;
+void EnableCustomServerCertificateError()
+{
+  // Safeguarding the handler when unsupported runtime is used.
+  try
+  {
+    if (!_isServerCertificateError)
+    {
+      webView.CoreWebView2.ReceivingServerCertificateError += WebView_ReceivingServerCertificateError;
+    }
+    else
+    {
+      webView.CoreWebView2.ReceivingServerCertificateError -= WebView_ReceivingServerCertificateError;
+    }
+    _isServerCertificateError = !_isServerCertificateError;
+
+    MessageBox.Show(this,
+      _isServerCertificateError ? "Custom server certificate error has been enabled" : "Custom server certificate error has been disabled",
+      "Custom server certificate error");
+  }
+  catch (NotImplementedException exception)
+  {
+    MessageBox.Show(this, "Custom server certificate error Failed: " + exception.Message, "Custom server certificate error");
+  }
+}
+
+void WebView_ReceivingServerCertificateError(object sender, CoreWebView2ReceivingServerCertificateErrorEventArgs e)
+{
+  if (e.ErrorStatus == CoreWebView2WebErrorStatus.CertificateIsInvalid)
+  {
+    // Continue the request with the SSL certificate.
+    e.Handled = true;
+  }
+  else
+  {
+    // Cancel the request.
+    e.Cancel = true;
+  }
+}
+
+// This example clears the SSL decision in response to proceeding with SSL certificate errors.
+void ClearSSLDecision()
+{
+  webView.CoreWebView2.ClearSSLDecision();
+  MessageBox.Show(this, "Cleared", "Clear SSL Decision");
+}
+```
+
+# API Details
+
+## Win32 C++
+``` cpp
+[uuid(4B7FF0D2-8203-48B0-ACBF-ED9CFF82567A), object, pointer_default(unique)]
+interface ICoreWebView2_11 : ICoreWebView2_10 {
+  /// Add an event handler for the ServerCertificateError event.
+  /// The ServerCertificateError event is raised when the WebView2
+  /// cannot verify server's digital certificate while loading a web page.
+  ///
+  /// With this event you have several options for responding to server
+  /// certificate error requests:
+  ///
+  /// Scenario                                                   | Handled | Cancel
+  /// ---------------------------------------------------------- | ------- | ------
+  /// Ignore the warning and continue the request                | True    | False
+  /// Cancel the request                                         | n/a     | True
+  /// Display default SSL interstitial page                      | False   | False
+  ///
+  /// If you don't handle the event, WebView2 will show the default SSL interstitial page to user.
+  ///
```

Comment:
You should include info about how ignoring the error and continuing the request is cached. I guess that ReceivingServerCertificateError won't be raised again for that certificate if you set Handled true? Is that cached for just the session or forever until ClearSslDecision is called or something else?